workflows/execute_workflows.sh

```diff
@@ -42,17 +42,27 @@ save_workspaces() {
     mv "$WORKSPACE_DIR"/"$2".zip "$WORKFLOW_DIR"/results/"$2"_"$WORKFLOW_NAME".zip
 }
 
-mkdir "$WORKSPACE_DIR"
-mkdir -p "$WORKFLOW_DIR/nf-results"
+source "$ROOT"/venv/bin/activate
 
 echo "Initialize submodules …"
 git submodule update --init --recursive
```

Comment:
```suggestion
git -C submodules/quiver-data submodule update --init
git -C submodules/oton submodule update --init
```

It's sufficient to update only the submodules needed for this feature.